torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -624,7 +624,7 @@ ProcessGroupNCCL::ProcessGroupNCCL(
       "ProcessGroupNCCL is only supported with GPUs, no GPUs found!");
   blockingWait_ = parseEnvVarFlag(NCCL_BLOCKING_WAIT);
   asyncErrorHandling_ =
-      static_cast<ErrorHandlingMode>(parseEnvVarInt(NCCL_ASYNC_ERROR_HANDLING));
+      static_cast<ErrorHandlingMode>(parseEnvVarIntDefault(NCCL_ASYNC_ERROR_HANDLING, 0));
```

Comment:
Just curious, right not when it is not set, where does it default to value -2?

Reply:
Here: https://github.com/pytorch/pytorch/blob/master/torch/csrc/distributed/c10d/Utils.hpp#L88